src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,56 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    public static final int INDEX_OF_OPERATION = 0;
+    public static final int INDEX_OF_SUM_OPERATION = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeDate(report(readFile(fromFileName).replaceAll("[\\[|\\]]","")), toFileName);
+    }
+
+    private String readFile(String fromFileName) {
+        File file = new File(fromFileName);
+        List<String> stringList;
+        try {
+            stringList = Files.readAllLines(file.toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("You got IOExeption", e);
+        }
+        return stringList.toString();
+    }
 
+    private String report(String date) {
```

Comment:
```suggestion
    private String buildReport(String date) {
```
method names should start with verb